tests/data/almanack/entropy/add_data.py

```diff
@@ -9,56 +9,67 @@
 from .add_entropy import insert_entropy
 
 
-def commit_changes(directory: str, message: str) -> None:
+def commit_changes(repo_path: pathlib.Path, message: str) -> None:
     """
-    Commits changes in the specified Git directory with a given commit message.
+    Commits changes in the given repository path with the provided commit message.
 
     Args:
-        directory (str): The directory containing the Git repository.
+        repo_path (pathlib.Path): The path to the Git repository.
         message (str): The commit message.
     """
-    repo = git.Repo(directory)
-    repo.git.add(".")
+    repo = git.Repo(repo_path)
+    repo.git.add(A=True)
     repo.index.commit(message)
 
 
 def create_repositories(base_path: pathlib.Path) -> None:
     """
     Sets up Git repositories with baseline content and adds entropy.
-    Repositories Structure:
-        High_Entropy
-            - high_entropy.md
-                - Baseline Content
-                - Added Entropy
-            - high_entropy2.md
-                -Baseline content
-        Low_Entropy
-            -Low_entropy.md
-                - Baseline content
-                - Added entropy
+
+    Args:
+        base_path (pathlib.Path): The base path where repositories will be created.
+
+    Notes:
+        Repository Structure:
+
+            Test_repo_1:
+                - File_1.md (Lines Changed: 61)
+                - File_2.md (Lines Changed: 31)
+                - File_3.md (Lines Changed: 16)
+            Test_repo_2:
+                - File_1.md (Lines Changed: 16)
+
+        The baseline content is committed first, followed by the addition of entropy.
```

Comment:
Aligning with the sentence below.
```suggestion
        The baseline content is committed first, followed by additional lines of changes.
```